metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -37,3 +37,60 @@ spec:
       for: 15s
       labels:
         severity: critical
+  - name: mirroring-alert.rules
+    rules:
+    - alert: OcsMirrorDaemonStatus
+      annotations:
+        description: Mirror daemon is in unhealthy status for more than 1m. Mirroring
+          on this cluster is not working as expected. For more info check Deployments
+          "rook-ceph-rbd-mirror-*" under openshift-storage namespace.
+        message: Mirror daemon is in unhealthy status.
+        severity_level: error
+        storage_type: ceph
+      expr: |
+        group by(namespace) (ocs_mirror_daemon_status{job="ocs-metrics-exporter"}==2) and (sum by(namespace) (ocs_pool_mirroring_status{job="ocs-metrics-exporter"}) > 0) > 0
+      for: 1m
+      labels:
+        severity: critical
+    - alert: OcsPoolMirroringImageHealth
+      annotations:
+        description: Mirroring image(s) (PV) under a pool {{ $labels.name }} are in
+          Unknown state for more than 1m. Mirroring might not work as expected. Please
+          contact support.
+        message: Mirroring image(s) (PV) under a pool {{ $labels.name }} are in Unknown
+          state.
+        severity_level: warning
+        storage_type: ceph
+      expr: |
+        group by(name, namespace)((ocs_pool_mirroring_image_health{job="ocs-metrics-exporter"} == 1) and  (ocs_pool_mirroring_status{job="ocs-metrics-exporter"}) == 1) > 0
+      for: 1m
+      labels:
+        severity: warning
+    - alert: OcsPoolMirroringImageHealth
+      annotations:
+        description: Mirroring image(s) (PV) under a pool {{ $labels.name }} are in
+          Warning state for more than 1m. Mirroring might not work as expected. Please
+          contact support.
+        message: Mirroring image(s) (PV) under a pool {{ $labels.name }} are in Warning
+          state.
+        severity_level: warning
+        storage_type: ceph
+      expr: |
+        group by(name, namespace)((ocs_pool_mirroring_image_health{job="ocs-metrics-exporter"} == 2) and  (ocs_pool_mirroring_status{job="ocs-metrics-exporter"}) == 1) > 0
+      for: 1m
+      labels:
+        severity: warning
```

Comment:
This looks like a duplicate.

Reply:
I am creating alerting rule for:
   1. Unknown
   2. Warning
   3. Error
   
   3 rules for the image health alert
   
   (ocs_pool_mirroring_image_health{job="ocs-metrics-exporter"} == 1 Warning
   (ocs_pool_mirroring_image_health{job="ocs-metrics-exporter"} == 2  Warning
   (ocs_pool_mirroring_image_health{job="ocs-metrics-exporter"} == 3   Critical
   
   
   alerting message tells exact health like unknown or warning or error